consensus/README.md

```diff
@@ -140,6 +140,222 @@ graph TB
   J --> note2
 ```
 
+## Synchronization between the consensus processes
```

Comment:
Make sure to install [1] https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one so that the table of contents is updated on save.

[1]